chapters/image_accceses.md

```diff
@@ -0,0 +1,97 @@
+# Image Accesses
+
+SPIR-V has various [Image Instructions](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#_image_instructions) that will interact with `OpTypeImage`. This chapter aims to give some extra context around these instructions.
+
+> There is an [example.frag](./examples/image_accesses/example.frag) (and [example.spv](./examples/image_accesses/example.spv)) GLSL shader that has an example of various ways images can be accessed in SPIR-V.
+
+# Images as Handles
+
+An `OpTypeImage` can be thought of as a handle the underlying data. To help illustrate this, here is a small program that writes to an image
+
+```glsl
+layout(set = 0, binding = 1, rgba8) uniform writeonly image2D image;
+void main() {
+    imageStore(image, ivec2(0, 0), vec4(1.0));
+}
+```
+
+Taking a closer look at the `OpVariable` we see it is of type [UniformConstant](../chapters/storage_class.md#uniformconstant) which is suppose to be "read-only" (yet there is a clear `OpImageWrite` occuring here).
+
+This is possible because the handle to the image is read-only, but the underlying texel data of the image is mutable.
+
+A software mental model of the `OpTypeImage` is to think of it as container that holds metadata while also holding the data of the image.
+
+```c++
+struct ImageType {
+  bool depth;
+  bool arrayed;
+  bool sampled;
+  bool multisampled;
+  Format format;
+  int width;
+  // other metadata ...
+  void* texels; // data of the image (which is mutable)
+}
+```
+
+The above `OpVariable` is read-only because we are not allowed to adjust its format, width, or multisampling state ever.
+
+# Sampled Images
+
+When using a sampler, you must have a `OpTypeSampledImage` object, this will be directly pointing to a `OpTypeImage` object.
```

Comment:
Consider using "an" when talking about a single `Op...`. e.g.:

"When using a sampler, you must have *an* `OpTypeSampledImage` object..."